### ISSUE TITLE
Only 5 minutes to get into first blue circle?

### ISSUE THREAD

**Issue by fttfbass** [BattleRoyaleGames/ArmA3-BattleRoyale-Issues#19]
I don't know if this is intentional or possibly related to issue #8.

Every now and then, the first notification will pop up after 20 minutes of gameplay (as usual), but it will only give 5 minutes instead of 10 to get into the first circle.  If you're nowhere near the circle or a fast vehicle, you might as well disconnect or suicide.  I notice that when this does happen, their are a lot more deaths than usual within the first minute or two after play is restricted inside that first circle (and they don't say killed by friendly fire).  I first thought that maybe the time given was related to the current player count (less players left=less time to get into the first circle), but that doesn't look to be the case after playing a bunch of games where it happened.

**Comment by VeritechHT1** [comment]
I've seen the same issue recently when playing ArmA 2, DayZ BR. In regards to A3-BR, the change-log states that the initial blue circle should be smaller if there are less than 15 people when the first warning is announced: nothing about changing the time to get into the first circle however.

http://battleroyalegames.com/changelog.txt (look under section 'v0.2.0').

**Comment by VeritechHT1** [comment]
fttfbass, I just went back and looked at some of my video capture and found a match from 15 April 2014 where the first announcement was for five minutes instead of ten minutes: so it doesn't appear that it is a new problem. There has always been issues with the timing of events in the mod but these have primarily revolved around when event(s) start and the difference has been small (I.e. first bomb zone starts nine minutes into the match as opposed to ten minutes in; first 'circle announcement' being made nineteen minutes into the round as opposed to twenty minutes elapsed).

And in video capture that I looked at, there were only fourteen players left when the first announcement was made: I'll take a look to see if I have any more occurrences. Again, from what I can read in the change-log, the number of players left should only impact the size of the circle and not the amount of time allotted to moving into the circle. My guess would be that what the algorithm does is that it skips the 'first circle' if player count is less than fifteen to the second circle. But since you only have five minutes between first and second circle enforcement, this value carries over as well. Again, I don't know if this is intended or not. Should be a simple fix if so: I do think that a smaller first circle makes sense if the player count is low, but that it should still be ten minutes to get into the first circle as people are still looting gear in some cases.

Edit: changed 'fifteen players' to 'fourteen players' after watching video again.

**Comment by VeritechHT1** [comment]
Okay, I've gone back and looked at a few more video captures: I found another six occurrences but the pattern/theory I proposed doesn't seem to hold true. I.e. I found instances where there was only a five minute warning and there were more than fifteen people left on the server. And although it appears that a lot of the occurrences happen with 'early' announcements, there are other occasions when the announcement is made early and yet with the proper ten minute timer.

So at most, I can confirm with video capture that what fttfbass has articulated is more than a 'one-off' occurrence.

**Comment by fttfbass** [comment]
I just had 3 games in a row that only gave 5 minutes to get into the first blue circle.  I barely made it into the circle in the first 2 games and had absolutely no chance to get into the 3rd.

5 minutes isn't enough time, unless you hang around the middle of the map every single game.  Which doesn't give players much of an incentive to move around the map much.

**Comment by Uelmer** [comment]
I agree - 5 minutes are too short. The shrinking is too fast too. Plz give us a little longer time to get in the circle and decrease the shrinking of the circle. Right now it is just a game about running behind the blue circle. If there are other players - dont shoot them, because then you have no time to get into the circle in time. Plz check the time balancing / extend the time to get in the circle.
